features/package-install.feature

```diff
@@ -158,6 +158,7 @@ Feature: Install WP-CLI packages
     Then STDOUT should contain:
       """
       Removing require statement from {PACKAGE_PATH}composer.json
+      Removing repository details from {PACKAGE_PATH}composer.json
```

Comment:
This only tests that there was console output.

The test should also verify that the repository setting has effectively been removed from the `composer.json` file.

Reply:
Made the changes @schlessera